### DISCUSSION TITLE
DietPi-Software | PhotoPrism

### DISCUSSION THREAD

**Discussion by id4vip** [MichaIng/DietPi#5727] [Software Request]
Good software for photo management for normal user.
https://photoprism.app

**Comment by lukaszsobala** [comment]
This is a very impressive piece of software but probably there is no point running it on devices that have less than 4 GB of RAM. 

Also, it runs as a Docker container and therefore much consideration needs to be put into its configuration - where the container images, where the photoprim database is and where the original images are stored. If it is implemented in dietpi, the installation process probably should infor the user about these things.

However, there should be a way to run it on bare metal, without docker, no?
